fe4u-boilerplate-main/src/js/API/callApi.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+function callApi(apiURL, endpoint, method) {
+    const url = apiURL + endpoint;
+    const options = {
+        method: method,
+    };
+    return fetch(url, options)
+        .then((response) => (response.ok ? response.json() : Promise.reject(Error('Failed to load'))))
```

Comment:
You don't need to check `response.ok`. When it's missed, the error will be generated and caught automatically.